modules/ROOT/pages/afm-add-schedule.adoc

```diff
@@ -5,43 +5,57 @@ endif::[]
 
 :imagesdir: ../assets/images
 
-You can add a schedule to an existing monitor if either of the following two conditions is true:
+You can add a schedule to an existing monitor if either of the following two conditions are true:
 
-* You plan to run the test associated with the schedule in a public location and the allocation of schedules for your organization in Anypoint Platform is not exhausted.
+* You plan to run the test associated with the schedule in a public location, and the allocation of schedules for your Anypoint Platform organization has not been exhausted.
 * You plan to run the test associated with the schedule in a private location.
 
-When you add a schedule to a monitor, you can use the schedule to run any of the versions of the test for that monitor.
-
 == Procedure
 
-There are two different methods that you can use to add a schedule.
-
-To add a schedule from the *Schedules* section of a monitor:
-
 . Select the monitor in which you want to add a schedule.
-. Click the *Schedules* tab.
-. Click *Add Schedule*.
-. In the Add Schedule dialog, specify this information:
+. Click *Add Schedule* from the *Schedules* table.
+. In the Add Schedule dialog, specify the following:
 +
-* The test version that you want the schedule to run
-* The location to run the test from
-* The frequency at which to run the test
-. Click *Add*.
+* The location in which to run the test
+* The interval at which to run the test:
++
+|===
+|Interval |Required Information |Defintion
 
-To add a schedule from the list of schedules that use a particular location:
+|Hourly
+|Minutes
+|The number of minutes after each hour when the test runs, beginning 60 minutes after the schedule has been created
+
+|Daily
+|Time
+|The time (hour and minutes) in a day when the test runs
+
+|Weekly
+|Day of week, Time
+|The day of the week and time when the test runs
+
+|Monthly
+|Day of month, Time
+|The day of the month and time when the test runs
+
+|Advanced
+|Cron
+|The cron expression describing when the test runs
+
+|===
+
+* Set Starting at (min)
+* Set configuration
 
-. Display the list of locations by clicking the *Locations* tab at the top of the screen.
-. Select the location that you want to use in a new schedule.
-. Click *Add Schedule* under the list of schedules to the right of the list of locations.
-. In the Add Schedule dialog, specify this information:
-+
-* The name of the monitor to apply the schedule to
-* The test version that you want the schedule to run
-* The frequency at which to run the test
 . Click *Add*.
 
 == Result
-The countdown to the first run of the selected test version begins.
+The countdown to the first run of the selected test begins.
```

Comment:
```suggestion
The 60-minute countdown to the first run of the selected test begins.
```

Reply:
The 60-minute countdown only applies if the user selected the Hourly monitor schedule, and this info is contained in the Hourly interval table row.